command_mode.c

```diff
@@ -2585,7 +2595,9 @@ struct command commands[] = {
 	{ "right-view",            cmd_right_view,       0, 0,  NULL,                 0, 0          },
 	{ "pl-create",             cmd_pl_create,        1, -1, NULL,                 0, 0          },
 	{ "pl-export",             cmd_pl_export,        1, -1, NULL,                 0, 0          },
+	{ "pl-save",               cmd_pl_save,          0, 0,  NULL,                 0, 0          },
```

Comment:
I think it is better to have the commands (as well as the docs) keep the alphabetic order inside their sections.